hdfs-scheduler/src/main/java/org/apache/mesos/hdfs/scheduler/HdfsNode.java

```diff
@@ -210,12 +209,17 @@ protected boolean enoughResources(Offer offer, double cpus, int mem) {
   private List<Task> createTasks(Offer offer) {
     String executorName = getExecutorName();
     String taskIdName = String.format("%s.%s.%d", name, executorName, System.currentTimeMillis());
-    List<Task> tasks = new ArrayList<Task>();
```

Comment:
For now line looks like `List<Task> tasks = new ArrayList<>();`
Got it. So you are asking about why I removed "Task" from "new ArrayList<Task>".
Afaik, we are using Jdk 1.7, supporting omitted type-parameter in the right part. 
So that could be simplified.